Source/Diagnostics/ReducedDiags/FieldProbe.cpp

```diff
@@ -385,6 +388,11 @@ void FieldProbe::ComputeDiags (int step)
     {
         const amrex::Geometry& gm = warpx.Geom(lev);
         const auto prob_lo = gm.ProbLo();
+        amrex::Real const dt = WarpX::GetInstance().getdt(lev);
+        if (do_moving_window_FP)
```

Comment:
See comment in header file:
```suggestion
        // Calculates particle movement in moving window sims
        amrex::Real move_dist = 0.0;
        bool const update_particles_moving_window =
            do_moving_window_FP &&
            step > warpx.start_moving_window_step &&
            step <= warpx.end_moving_window_step;
        if (update_particles_moving_window)
```